Week-3/Exercise-3.1/script.js

```diff
@@ -0,0 +1,22 @@
+//Given reducer method:
+function add(a, b) {
```

Comment:
We should try to use ES6 features as much as we can. Use arrow functions here

Reply:
resolved and committed